### ISSUE TITLE
add method in linklist does not work properly

### ISSUE THREAD

**Issue by Mangesh-Deshpande** [sagar-gavhane/rahome#32]
when i create a linklist and add some elements to it the result is wrong.Please refer the following code to re-create the issue.

```javascript
const { LinkedList } = require("rahome");

const linklist = new LinkedList();

linklist.add(10);
linklist.add(20);
linklist.add(30);
linklist.add(40, 4);

console.log(linklist.toString());

output - 10->40

expected result - 10->20->30->40
```

**Comment by sagar-gavhane** [comment]
Thanks, @Mangesh-Deshpande for highlighting this issue.

**Comment by sagar-gavhane** [comment]
This issue has been fixed in #33 PR.
